api/src/main/java/com/redhat/insights/http/BackoffWrapper.java

```diff
@@ -21,6 +21,12 @@ public interface Action {
     void run() throws Throwable;
   }
 
+  public static class FatalBackoffException extends RuntimeException {
```

Comment:
What is this exception for? It doesn't seem to be used anywhere.

Reply:
Sorry this code was _NOT_ supposed to be there!